packages/fxa-auth-server/config/index.ts

```diff
@@ -857,14 +857,6 @@ const conf = convict({
       env: 'SUBSCRIPTIONS_ENABLED',
       default: false,
     },
-    coupons: {
-      enabled: {
-        doc: 'feature flag for enabling coupons',
-        default: false,
-        env: 'COUPONS_ENABLED',
-        format: Boolean,
-      },
-    },
```

Comment:
Huh; I guess this was never used anywhere after it was added in #11048 . 😅 At least we separately blocked coupons in the front-end via the other flag.